src/Apps/Artwork/Components/__tests__/ArtistInfo.test.tsx

```diff
@@ -22,9 +22,13 @@ describe("ArtistInfo", () => {
       },
     })
   }
+  let wrapper
 
-  it("renders a correct component tree", async () => {
-    const wrapper = await getWrapper()
+  beforeAll(async () => {
+    wrapper = await getWrapper()
+  })
```

Comment:
Since the first test case below is an outlier, I don't think we should wrap this in a beforeAll -- if _most_ tests didn't need to be booted with data then it would make sense.